scripts/style_api_gen.py

```diff
@@ -94,6 +94,10 @@
   'style_type': 'num',   'var_type': 'int32_t',  'default':0, 'inherited': 0, 'layout': 1, 'ext_draw': 1,
   'dsc': "Skew Widget vertically. The value is interpreted in 0.1 degree units. E.g. 450 means 45 deg."},
 
+{'name': 'TRANSFORM_MATRIX',
+ 'style_type': 'ptr',   'var_type': 'const lv_matrix_t *' ,  'default':'`NULL`', 'inherited': 0, 'layout': 0, 'ext_draw': 0,
+ 'dsc': "An initialized `lv_matrix_t` to describe a matrix."},
```

Comment:
Mentioned that it requires `LV_USE_MATRIX` and GPU which supports matrix transformations.

Reply:
Done